lib/polyfill/media_capabilities.js

```diff
@@ -27,7 +27,12 @@ shaka.polyfill.MediaCapabilities = class {
     // Since MediaCapabilities is not fully supported on Chromecast yet, we
     // should always install polyfill for Chromecast.
     // TODO: re-evaluate MediaCapabilities in the future versions of Chromecast.
-    if (!shaka.util.Platform.isChromecast() && navigator.mediaCapabilities) {
+    // Since MediaCapabilities is not fully supported on Apple browsers yet, we
```

Comment:
This comment should probably say that we are installing the polyfill on Apple browsers because their implementation is buggy, not that it is incomplete.
Linking to https://github.com/google/shaka-player/issues/3530 to give the comment context would be a good idea too.

Reply:
Done!